common/sepolicy/vendor/file_contexts

```diff
@@ -1,3 +1,19 @@
+#
+# Copyright (C) 2020 Roman Stratiienko (r.stratiienko@gmail.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
This file has been copy-pasted from AOSP

Reply:
I've changed the copyright

Reply:
The original file had no copyright notice. Please do not put any.

Reply:
Sure, I dropped for both files